gst/nnstreamer/nnstreamer_conf.c

```diff
@@ -34,6 +34,12 @@ static const gchar *subplugin_prefixes[] = {
   [NNSCONF_PATH_END] = NULL
 };
 
+#ifdef G_OS_WIN32
+static const gchar root_path_prefix[] = "c:\\";
```

Comment:
In case of windows,  is "C:\\"  reasonable to all users?

Reply:
Let's not think too much for WINDOWS at this point. We don't use "etc" in Windows either.